src/studio/src/designer/frontend/shared/package.json

```diff
@@ -24,6 +24,7 @@
     "react-content-loader": "~6.0.3",
     "react-html-parser": "~2.0.2",
     "react-monaco-editor": "~0.43.0",
+    "react-redux": "~7.2.4",
```

Comment:
Is this intentional? Do we use redux in our shared project?

Reply:
Yes, line 2 in: shared/features/dataModelling/DataModelling.tsx